ddtrace/tracer/option.go

```diff
@@ -510,6 +523,17 @@ func WithEnv(env string) StartOption {
 	}
 }
 
+// WithServiceMapping holds a mapping of services used to dynamically rename services.
+// This option is is case sensitive and can be used multiple times.
+func WithServiceMapping(k string, v string) StartOption {
```

Comment:
```suggestion
// WithServiceMapping determines service "from" to be renamed to service "to".
// This option is is case sensitive and can be used multiple times.
func WithServiceMapping(from, to string) StartOption {
``` 
Note that you are using the word "holds", which is perhaps appropriate for a variable or constant. A function _does_ something; it does not _hold_ something.

Also, take care when writing documentation. Notice my version, which is (IMHO) a simple explanation, and your version, which is quite complex and confusing. It's best to put yourself in the user's shoes. Always try to keep it as simple and "dumb" of an explanation as possible, leaving no ambiguity. The goal is for things to be easy to use and understand. It's also a good practice to consider the names of variables and use those names in the documentation (see what you had and what I proposed).

Reply:
Thanks for the feedback, I will make sure to be more thoughtful when writing documentation in the future! Noted the tips about keeping the explanation simple and using the name of variables.

Reply:
Made this change 👍